Mathlib/RingTheory/Polynomial/Hermite/Basic.lean

```diff
@@ -0,0 +1,150 @@
+/-
+Copyright (c) 2023 Luke Mantle. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Luke Mantle
+
+! This file was ported from Lean 3 source module ring_theory.polynomial.hermite.basic
+! leanprover-community/mathlib commit 066ecdb4834c7a4693e0f0e5154935a6f3d3f90c
+! Please do not edit these lines, except to modify the commit id
+! if you have ported upstream changes.
+-/
+import Mathlib.Data.Polynomial.Derivative
+import Mathlib.Data.Nat.Parity
+
+/-!
+# Hermite polynomials
+
+This file defines `Polynomial.hermite n`, the nth probabilist's Hermite polynomial.
+
+## Main definitions
+
+* `Polynomial.hermite n`: the `n`th probabilist's Hermite polynomial,
+  defined recursively as a `Polynomial ℤ`
+
+## Results
+
+* `Polynomial.hermite_succ`: the recursion `hermite (n+1) = (x - d/dx) (hermite n)`
+* `Polynomial.coeff_hermite_of_odd_add`: for `n`,`k` where `n+k` is odd, `(hermite n).coeff k` is
+  zero.
+* `Polynomial.monic_hermite`: for all `n`, `hermite n` is monic.
+
+## References
+
+* [Hermite Polynomials](https://en.wikipedia.org/wiki/Hermite_polynomials)
+
+-/
+
+
+noncomputable section
+
+open Polynomial
+
+namespace Polynomial
+
+/-- the nth probabilist's Hermite polynomial -/
```

Comment:
```suggestion
/-- the probabilists' Hermite polynomials. -/
```
We're not talking about the n'th probabilist, and there should be a full stop (I know this is what the mathlib3 doc said but may as well change it here). If you're happy with this suggested change then you could also change the module docstring. I think "the n'th probabilist" sounds very odd.